cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -54,7 +55,7 @@ def __init__(
         self.housekeeper = hk_api
         self.scout_api = scout_api
         self.madeline_api = madeline_api
-        self.mip_analysis_api = analysis_api
+        self.analysis_api = analysis_api
```

Comment:
Have you double-checked all usages of this attribute? Renaming things can be dangerous

Reply:
Yes, indeed it is tricky to change that, restoring